test/functional/lcow_container_bench_test.go

```diff
@@ -0,0 +1,228 @@
+//go:build windows && functional
+// +build windows,functional
+
+package functional
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	ctrdoci "github.com/containerd/containerd/oci"
+	cri_util "github.com/containerd/containerd/pkg/cri/util"
+
+	"github.com/Microsoft/hcsshim/internal/hcsoci"
+	"github.com/Microsoft/hcsshim/internal/resources"
+	"github.com/Microsoft/hcsshim/osversion"
+
+	"github.com/Microsoft/hcsshim/test/internal/cmd"
+	"github.com/Microsoft/hcsshim/test/internal/constants"
+	"github.com/Microsoft/hcsshim/test/internal/container"
+	"github.com/Microsoft/hcsshim/test/internal/containerd"
+	"github.com/Microsoft/hcsshim/test/internal/layers"
+	"github.com/Microsoft/hcsshim/test/internal/oci"
+	"github.com/Microsoft/hcsshim/test/internal/require"
+	"github.com/Microsoft/hcsshim/test/internal/uvm"
+)
+
+func BenchmarkLCOW_Container(b *testing.B) {
+	requireFeatures(b, featureLCOW, featureContainer)
+	require.Build(b, osversion.RS5)
+
+	ctx, _, client := newContainerdClient(context.Background(), b)
+	chainID := containerd.PullImage(ctx, b, client, constants.ImageLinuxAlpineLatest, constants.PlatformLinux)
+	ls := layers.FromChainID(ctx, b, client, chainID, constants.SnapshotterLinux)
```

Comment:
might be worth moving this to a helped function as well

Reply:
do you mean a function that pulled an image then returned the layers instead of the chainID?